persistence/src/vespa/persistence/spi/package.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Was adding this file intentional?

Reply:
No certainly not. I wonder if this is something clion has generated.....
https://github.com/vespa-engine/vespa/pull/16080